eatin-server/src/graphql/userRecipes/userRecipes.dto.ts

```diff
@@ -1,5 +1,8 @@
+import { Recipes } from '../recipes/recipes.model';
+
 export interface UserrecipeDTO {
```

Comment:
why not removing recipe_index and always return the recipe?

Reply:
then why did you add the whole recipe and not just the recipe_index?